python/validate_tdr_file_export.py

```diff
@@ -0,0 +1,54 @@
+import csv
+from google.cloud import storage
+import base64
+import binascii
+from argparse import ArgumentParser, Namespace
+from utils.tdr_utils.tdr_api_utils import TDR
+from utils.request_util import RunRequest
+from utils.token_util import Token
+
+
+def get_args() -> Namespace:
+
+    parser = ArgumentParser(
+        description="""validation of TDR file export to GCP bucket""")
+
+    parser.add_argument("-id", "--dataset_id", required=True,
+                        help="ID of dataset from export")
+    parser.add_argument("-b", "--bucket_id", required=True,
+                        help="Google bucket to check")
+    parser.add_argument("-o", "--output_file", required=False,
+                        default="~/dataset_export_validation.csv",
+                        help="Output file for validation results")
+    return parser.parse_args()
+
+
+if __name__ == "__main__":
+    args = get_args()
+    token = Token(cloud='gcp')
```

Comment:
Could we import GCP from util __init__ instead of using string?

Reply:
updated to use util init here.